src/MovieApp.java

```diff
@@ -0,0 +1,211 @@
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class MovieApp {
+
+    Movie[] movies = new Movie[0];
+
+    public void press() {
+        boolean isRun = true;
+        while (isRun) {
+            showMenu();
+            System.out.println("Press the movie button: ");
+            int num = new Scanner(System.in).nextInt();
+
+            switch (num) {
+                case 0:
+                    isRun = false;
+                    break;
+                case 1:
+                    inputMovies();
+                    break;
+                case 2:
+                    displayMovies(movies);
+                    break;
+                case 3:
+                    addMovies();
+                    break;
+                case 4:
+                    findStatistics();
+                    break;
+                case 5:
+                    searchMovie();
+                    break;
+                case 6:
+                    updateRating();
+                    break;
+                case 7:
+                    deleteMovie();
+                    break;
+                case 8:
+                    sortMovie();
+                    break;
+                default:
+                    System.out.println("Invalid input");
+                    break;
+
+            }
+            if (!isRun) break;
+        }
+
+    }
+
+    public static void showMenu() {
+        System.out.println("""
+                Press 1: to input movies
+                Press 2: to display movies
+                Press 3: to add movies
+                Press 4: to find statistics
+                Press 5: to search movie
+                Press 6: to update ratings
+                Press 7: to delete movies
+                Press 8: to sort movies
+                Press 0: to exit
+                """
+        );
+    }
+
+    private void inputMovies() {
+        movies = new Movie[3];
```

Comment:
try to do logic like that, we can reuse this method, I mean if I have 10 Movies and I press 1 my movies will be 3, but wanted to be 13